Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,30 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_renewable_energy",
+            category=['technology'],
+            prompt="Renewable Energy",
+            random=True
+        )
+    )
+
+label monika_renewable_energy:
+    m 4eub "What do you think about renewable energy, [player]?"
+    m 4eud "Honestly, I care about it a lot."
+    m 7rsd "As humanity's reliance on technology grows, so too does its demand for energy."
+    m 1esd "That's fine on its own, but then fossil fuels come in."
+    m 1ekc "Currently, a signifigant percentage of energy is produced from burning fossil fuels."
+    m 2ekc "Part of the problem is that renewable energy isn't without its drawbacks."
+    m 3ekc "Take hydropower, for example."
+    m 5lkc "Hydropower is flexible, efficient, and hardly produces any carbon dioxide, but it can also severely affect the local area."
```

Comment:
Dissolve into body pose 5. 

We also typically reserve leaning poses for more romantic moments.

Reply:
I actually think leaning (or head tilting) works well here as a 'pondering' or 'thinking' pose. It kind of reminds me of Palutena from Kid Icarus: Uprising:

![image](https://user-images.githubusercontent.com/71156445/93000228-ed615d00-f4f4-11ea-951a-81b43be1759c.png)

I can probably find other poses if you want, but I think these work well to give off the aura that she's thinking it over as she speaks.

Reply:
![image](https://user-images.githubusercontent.com/45746485/93000555-8c07b100-f529-11ea-9f51-00c69ccf2771.png)

This one is a very common "Monika is thinking" pose I use. Maybe you'll like a variant of it.

Reply:
Thank you, I'll take a look at those once I've finished rewriting the dialogue.